### ISSUE TITLE
Slider in Preferences to adjust reisze speed

### ISSUE THREAD

**Issue by boltronics** [gnome-terminator/terminator#560]
**Is your feature request related to a problem? Please describe.**
I regularly use the `resize_<direction>` keybindings. They're great and intuitive for terminal resizing operations, but sometimes they feel too slow to make big adjustments. I often find myself reaching for the mouse to perform a resize to speed up the process. Holding down one of the keybindings only seem to make size adjustments of around 8-10 lines per second, so can be a few second wait on a big 4K screen with potentially hundreds of lines to shrink/expand by.

**Describe the solution you'd like**
It would be great if there were a slider in the Global Terminator Preferences to adjust the resize speed when holding down the shortcut bound to these resize operations.

**Describe alternatives you've considered**
I've looked to how Konsole manages it, but Terminator is already leagues ahead in this department. Aside from Konsole offering no options regarding the direction it will shrink/expand, there are also no options to adjust resize speed.

However, Konsole seems to adjust the speed during a resize operation automatically depending upon how big the terminal in question is at any given moment. If the terminal is small, Konsole will make resize adjustments in tiny increments. If it's large, it adjusts by huge increments. I personally find this approach to be problematic as it's impossible to get the large size readjustments exactly as desired unless doing so with the mouse, rendering the keybinding partially useless. It reminds me of the way mouse acceleration works, which I also strongly dislike.

In short, Terminator's fixed "line-at-a-time" approach is much nicer IMO, but it just needs an option to speed it up. Even should a very fast speed be selected, there will be that momentary delay before the repeat operation kicks in from holding the shortcut down. Hence, for my proposal where the user may set a slider to an extremely fast speed, one would be able to make big adjustments quite quickly by holding down the shortcut (but possibly a bit inaccurate depending on the user's reflexes), and if necessary, finish up by simply tapping the arrow keys normally to dial in the exact size preference.

**Comment by jbericat** [comment]
Any news about this? I'd really love to see this feature request implemented
